src/carbonio-files-ui-common/views/components/sharing/publicLink/PublicLink.tsx

```diff
@@ -33,12 +33,16 @@ interface AddPublicLinkProps {
 	nodeId: string;
 	nodeName: string;
 	canShare: boolean;
+	linkName: string;
+	linkDescription: string;
 }
 
 export const PublicLink = ({
 	nodeId,
 	nodeName,
-	canShare
+	canShare,
```

Comment:
I think we should remove this prop and conditionally render the PublicLink component where it is used instead of returning an empty container here. Doing so, we could avoid requesting the data of the links. @cmazzilli what do you think?